src/Core/Domain/Common/Events/EntityDeletedEvent.cs

```diff
@@ -0,0 +1,9 @@
+namespace FSH.WebApi.Domain.Common.Events;
+
+public class EntityDeletedEvent<T> : DomainEvent
```

Comment:
Is this more like a generic event?

Reply:
Yes, this way you don't have to define each event for each entity.